### ISSUE TITLE
Problem after creating a new table attribute

### ISSUE THREAD

**Issue by KryptoKnight** [flagbit/akeneo-table-attribute-bundle#40]
Hello, 

First of all let me say thank you for taking the time to release and maintain this bundle.

Secondly, I'd like to outline the problem I'm having...

I recently installed a fresh version of Akeneo (3.2.12 CE) and then on top of this installation I installed the akeneo-table-attribute-bundle - visually everything looked good.

I then created by first table attribute and clicked save to see the "columns" option display... this also looked good.

The issues came when I started to add column values - upon clicking save I see a red alert icon which has a tooltop saying "500". and in the logs I have the following error:

`[2019-10-10 15:42:55] request.INFO: Matched route "pim_enrich_attributeoption_create". {"route":"pim_enrich_attributeoption_create","route_parameters":{"_controller":"pim_enrich.controller.attribute_option:createAction","_format":"json","attributeId":"11","_route":"pim_enrich_attributeoption_create"},"request_uri":"https://pim.example.com/configuration/attribute-option/11","method":"POST"} []
[2019-10-10 15:42:55] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalThrowableError: "Call to a member function setValue() on null" at /var/www/pim.example.com/vendor/doctrine/orm/lib/Doctrine/ORM/PersistentCollection.php line 167 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Call to a member function setValue() on null at /var/www/pim.example.com/vendor/doctrine/orm/lib/Doctrine/ORM/PersistentCollection.php:167)"} []`

Any help or pointers would be appreciated.

Many Thanks,

**Comment by scivray** [comment]
Hello, 
Nearly same error with version pim v3.0.49.
I create a new attribute but when I click on Columns tab I got error 500 with this in logs : 

[2019-10-25 14:48:48] request.INFO: Matched route "pim_enrich_attributeoption_index". {"route":"pim_enrich_attributeoption_index","route_parameters":{"_controller":"pim_enrich.controller.attribute_option:indexAction","_format":"json","attributeId":"207","_route":"pim_enrich_attributeoption_index"},"request_uri":"https://mycompany:port/configuration/attribute-option/207","method":"GET"} []
[2019-10-25 14:48:48] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalThrowableError: "Call to a member function setValue() on null" at /srv/app/vendor/doctrine/orm/lib/Doctrine/ORM/PersistentCollection.php line 168 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Call to a member function setValue() on null at /srv/app/vendor/doctrine/orm/lib/Doctrine/ORM/PersistentCollection.php:168)"} []
[2019-10-25 14:48:48] request.CRITICAL: Exception thrown when handling an exception (RuntimeException: Failed to start the session because headers have already been sent by "/srv/app/vendor/doctrine/orm/lib/Doctrine/ORM/Persisters/Entity/BasicEntityPersister.php" at line 1773. at /srv/app/vendor/symfony/symfony/src/Symfony/Component/HttpFoundation/Session/Storage/NativeSessionStorage.php line 141) {"exception":"[object] (RuntimeException(code: 0): Failed to start the session because headers have already been sent by \"/srv/app/vendor/doctrine/orm/lib/Doctrine/ORM/Persisters/Entity/BasicEntityPersister.php\" at line 1773. at /srv/app/vendor/symfony/symfony/src/Symfony/Component/HttpFoundation/Session/Storage/NativeSessionStorage.php:141)"} []

**Comment by allanhansen-vaimo** [comment]
Same issue as @scivray.

Running Akeneo CE v. 3.2.31. Fresh install of both Akeneo CE and Flagbit table attribute.

**Comment by allanhansen-vaimo** [comment]
@amansilla any updates on this?

> Same issue as @scivray.
> 
> Running Akeneo CE v. 3.2.31. Fresh install of both Akeneo CE and Flagbit table attribute.

**Comment by onlyapproved** [comment]
We are having the same issue as @scivray and @allanhansen-vaimo

**Comment by onlyapproved** [comment]
@allanhansen-vaimo  @scivray  -> i found the issue being the apache service after installing the module. It's probably due to us running Akeneo from a Bitnami image. 

Restart apache service, and then it should be all ok.

**Comment by flagbird** [comment]
@scivray @allanhansen-vaimo Can you verify that this issue depends on the environment?

**Comment by allanhansen-vaimo** [comment]
> @scivray @allanhansen-vaimo Can you verify that this issue depends on the environment?

Yes, but it isn't related to Bitnami, it is also necessary to restart Apache and PHP-FPM containers in the Docker env. that Akeneo supplies.
